code/__DEFINES/supermatter.dm

```diff
@@ -132,3 +132,9 @@
 #define POWERLOSS_LINEAR_RATE 0.83
 /// How much a psychologist can reduce power loss.
 #define PSYCHOLOGIST_POWERLOSS_REDUCTION 0.2
+
+#define SUPERMATTER_COLOUR "#ffd04f"
+#define SUPERMATTER_RED "#aa2c16"
+#define SUPERMATTER_CASCADE_COLOUR "#333333"
+#define SUPERMATTER_TESLA_COLOUR "#00ffff"
+#define SUPERMATTER_SINGULARITY_COLOUR "[pick("#500000", "#400060")]"
```

Comment:
this seems kinda sus not gonna lie with you

Reply:
uh uh magic numbers avoiding right?

Reply:
the pick. please don't make macros do things like this

Reply:
it's a bit weird to have a color define that is not in fact one color
Also, I don't understand why the pick is wrapped with "[]"

Reply:
finish this review already!!!!!!!!!!!!!!